include/bl_validation.h

```diff
@@ -18,6 +18,21 @@
 #include <stdbool.h>
 #include <fw_info.h>
 
+/**Function for validating firmware in place.
```

Comment:
nit: Do we normally have a space here?  Ignore if no other change is needed.
```
/** Something
```

Reply:
I think yours looks nicer and is more commonly used :). Thanks